lib/elixir/lib/kernel.ex

```diff
@@ -2805,9 +2805,10 @@ defmodule Kernel do
   ## Module names
 
   A module name can be any atom, but Elixir provides a special syntax which is
-  usually used for module names. What is called a module name is an upper case
-  character followed by any alphanumeric characters or `_` or `.`. This
-  identifier is equivilant to an atom prefixed by `Elixir.`. So in the
+  usually used for module names. What is called a module name is an _upper case
+  Latin letter_ followed by any number of _Latin letters_, _numbers_, _underscores_
```

Comment:
I think maybe we should use the term latin letters. People might think something like `DéjàVu` is valid (it is in Erlang). perhaps ASCII would be better?